server/controllers/usersController.js

```diff
@@ -1,9 +1,21 @@
-// CONTROLLER FOR USER MODEL
+const bcrypt = require("bcrypt");
 
-const db = require("../models/user");
+const connection = require("../dbConnection");
+const userSchema = require("../models/user");
 
-// Defining methods for the usersController
-module.exports = {
-  // define methods here
+const User = connection.model("User", userSchema);
 
+const SALT_ROUNDS = Number(process.env.SALT_ROUNDS);
+
+const create = async ({ email, password }) => {
+    const hashedPassword = await bcrypt.hash(password, SALT_ROUNDS);
+    const user = await User.create({ email, password: hashedPassword });
+    return user;
 };
+
+const findOneWithemail = async (email) => await User.findOne({ email });
```

Comment:
consistent camelCase would be findOneWithEmail

Reply:
Good catch, I just fixed it.